scripts/extract_download_logs.py

```diff
@@ -57,7 +80,32 @@
     "rp_start",
     "rp_end",
 ]
-OUTPUT_FILENAME = options.filename
+OUTPUT_FILENAME = args.filename
+
+
+def send_notify(
+    from_date: datetime.datetime,
+    to_date: datetime.datetime,
+    file: StringIO,
+    api_key: str = os.environ["NOTIFY_API_KEY"],
+    template_id: str = "196e5553-886c-40bd-ac9a-981a7868301b",
```

Comment:
This template ID is specific to your Notify account and will not work for anyone else, so should not be stored here

Reply:
@wjrm500 As this is specific to our template and use case and not designed to be generically re-usable. I think this is fine as-is. If we have a more generic case in future we can make this overridable.